internal/pkg/queue/enqueue.go

```diff
@@ -35,6 +35,8 @@ func (q *PersistentGroupedQueue) BatchEnqueue(items ...*Item) error {
 	}
 
 	itemsChan := make(chan *msg, len(items))
+	defer close(itemsChan)
```

Comment:
Linked with https://github.com/internetarchive/Zeno/pull/78#discussion_r1696210853

Reply:
Not needed, at all, see my response to https://github.com/internetarchive/Zeno/pull/78#discussion_r1696210853